purchases/src/main/kotlin/com/revenuecat/purchases/interfaces/RedeemRCBillingPurchaseListener.kt

```diff
@@ -0,0 +1,43 @@
+package com.revenuecat.purchases.interfaces
+
+import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI
+
+/**
+ * Interface to handle the redemption of a RevenueCat Billing purchase.
+ */
+@ExperimentalPreviewRevenueCatPurchasesAPI
+interface RedeemRCBillingPurchaseListener {
+    /**
+     * Result of the redemption of a RevenueCat Billing purchase.
+     */
+    enum class RedeemResult {
+        SUCCESS,
+        ERROR,
```

Comment:
I'm actually planning to make this a sealed class, to provide the CustomerInfo/PurchaseError. We also will want to add additional result cases like "Link expired" or something along those lines, so the developer can handle appropriately more easily, rather than parsing the `PurchasesError`.

Reply:
That makes sense!